package.json

```
@@ -8,6 +8,7 @@
     "start": "node ./bin/ungit",
     "test": "grunt test",
     "unittest": "grunt unittest",
+    "format": "prettier --write .",
```

I have tested it a little bit more and it seems that `./bin/credentials-helper` and `./bin/ungit` are not formatted.
Manually including them like this fixed it:

```suggestion
    "format": "prettier --write . ./bin/credentials-helper ./bin/ungit",
```

---

Is that because they are not `.js` files?
If so, what is the main reason for them to not have the `.js` ending?